README.md

```diff
@@ -148,10 +148,17 @@ Changes to `.js` files are handled by Node server([react-scripts](https://github
 
 ## Test
 
+### Backend unit tests
 ```bash
 make test
 ```
 
+### Backend integration tests
+1. Follow [these instructions](#Run-locally) to target existing OpenShift project (**Note:** the project must not contain existing mobile clients)
+2. Install [mocha](https://mochajs.org/) test framework globally: `npm install mocha -g`
+3. Run the tests: `mocha integration_tests`
```

Comment:
I also need to run `npm install` in `integration_tests` folder. So I think it might be better to install `mocha` as a dependency there and have a `npm test` command that will run the tests, rather than installing `mocha` globally. 

wdyt?

Reply:
fixed